src/components/List/ListLibrary.tsx

```diff
@@ -0,0 +1,28 @@
+import { ListIcon } from '@/public/index';
+
+type ListLibraryProps = {
+  avatarUrl?: string;
+  userName?: string;
+};
+
+export function ListLibrary({
+  avatarUrl = 'https://avatars.githubusercontent.com/u/137901354?v=4',
+  userName = 'marryfile',
+}: ListLibraryProps) {
+  const containerStyles =
+    'flex h-[7rem] w-[24.7rem] items-center justify-between rounded-t-[0.8rem] border-b border-gray-300 bg-primary-50 p-[2.0rem]';
+
+  const userInfoStyles = 'flex h-[3.0rem] w-fit items-center justify-around gap-[1rem]';
+
+  const imgStyles = 'h-[3rem] w-[3rem] rounded-full';
+
+  return (
+    <div className={containerStyles}>
+      <div className={userInfoStyles}>
+        <img className={imgStyles} src={avatarUrl} />
```

Comment:
```suggestion
  return (
    <div className={containerStyles}>
      <div className={userInfoStyles}>
        <Image width={30} hight={30} className='rounded-full' src={avatarUrl} />
```
`<Image>` 태그 안쓰신 이유가 있나용?

Reply:
작은 이미지라 최적화가 딱히 필요하지 않다고 생각해서 안썼습니다

Reply:
그래도 쓰는게 좋지 않을까용? 👀 api로 불러오는 이미지인데, 개인적으로는 next에서는 대부분을 img보다는 Image 컴포넌트 쓰는게 좋은걸로 알아서 안쓸이유가 없다고 생각합니다. 이건 회의때 얘기 한번 해보는게 좋겠어용